src/ontology/ro-edit.owl

```diff
@@ -3959,6 +3962,7 @@ InverseObjectProperties(obo:RO_0002439 obo:RO_0002458)
 
 # Object Property: obo:RO_0002440 (symbiotically interacts with)
 
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0002440 "whales symbiotically interact with the barnacles that attach to them")
```

Comment:
@diatomsRcool same here. You added the example in `symbiotically interacts with` instead of `symbiotrophically interacts with`.

Reply:
I don't understand this comment. The object property is symbiotically interacts with and that is what is said in the example

Reply:
ok, I think I dealt with it